Cirrious.FluentLayout/FluentLayout.cs

```diff
@@ -153,7 +154,7 @@ private void ThrowIfSecondItemAlreadySet()
 
         public IEnumerable<NSLayoutConstraint> ToLayoutConstraints()
         {
-            var constraint = NSLayoutConstraint.Create(
+            Constraint = NSLayoutConstraint.Create(
```

Comment:
I think it makes sense to include this member in `FluentLayout` as it is a 1 to 1 relation. 1 FluentLayout <-> 1 NSLayoutConstraint.
Also, without calling `ToLayoutConstraints()`this object is totally useless. So I assume it always be called thus `Constraint` member will always be set.
I understand your point, but I don´t see an easier (with lesser code) way to gain access to NSLayoutConstraints